core/issuers.js

```diff
@@ -67,10 +67,13 @@ Description must be truthy: ${description}`;
 
     // makePaymentTrait is defined in the passed-in configuration and adds
     // additional methods to corePayment
+    const makePaymentTraitIter = makePaymentTrait(corePayment, issuer);
+    const paymentTrait = makePaymentTraitIter.next().value;
     const payment = harden({
-      ...makePaymentTrait(corePayment, issuer),
+      ...paymentTrait,
```

Comment:
Should the traits be plural?  The `...` makes me think there might be more than one.

Reply:
I could be wrong but my understanding of `trait` was that a trait could encompass multiple methods.